### ISSUE TITLE
Image preloading

### ISSUE THREAD

**Issue by jayjung1018** [GraceCovenantChurch/gccweb#4]
Currently, images load AFTER the page is loaded, which is not ideal - angular should have a preloader for images.

**Comment by jayjung1018** [comment]
seems to be better...

**Comment by joeyzia** [comment]
I don't know how Jay solved it, but I was gonna ask if we could do something like this...

http://www.adaptive-images.com

Would this help with loading images faster? Especially when we display high resolution photos?

**Comment by austinEng** [comment]
Definitely a good idea. Facebook does something like this for their images. But probably an enhancement we don't need yet

**Comment by jayjung1018** [comment]
yeah i couldn't necessarily tell - i tried some preloading techniques i found online, but really all i ended up doing was compressing the images down so that they're not >1mb - this seemed to shrink the time from 1s to like 50-60ms. 

adaptive-images also was for php apps i think, and didn't really take the time to look for alternatives.

it's also very dependent on peoples' connections, but i agree with austin - we can save this for later.
